node/src/components/chainspec_loader/chainspec.rs

```diff
@@ -123,6 +125,10 @@ impl HighwayConfig {
         {
             warn!("Era duration is less than minimum era height * round length!");
         }
+
+        if self.minimum_round_exponent >= self.maximum_round_exponent {
+            warn!("Minimum round exponent is greater than or equal to maximum round exponent.  Minimum round exponent: {min}, Maximum round exponent: {max}", min = self.minimum_round_exponent, max = self.maximum_round_exponent);
```

Comment:
```rust
    "You can break string literals using a \
    backslash!"
```

Reply:
And actually, maybe we should just panic here if it's strictly greater?
On the other hand if it's equal, maybe that's fine: You might want to run a network with a fixed block rate.

Reply:
Fixed in https://github.com/CasperLabs/casper-node/pull/479/commits/12d3251cad27747c248568412894a5fc61524459 .  I've changed this to be a strict equality as you have suggested.